bigbluebutton-html5/imports/ui/components/user-list/chat-list-item/component.jsx

```diff
@@ -32,84 +32,69 @@ const propTypes = {
-    const isCurrentChat = chat.id === openChat;
-    const isSingleMessage = chat.unreadCounter === 1;
+  const linkPath = [PRIVATE_CHAT_PATH, chat.id].join('');
+  const isCurrentChat = chat.id === openChat;
+  const isSingleMessage = chat.unreadCounter === 1;
 
-    const linkClasses = {};
-    linkClasses[styles.active] = isCurrentChat;
+  const linkClasses = {};
+  linkClasses[styles.active] = isCurrentChat;
 
-    if (chat.name === 'Public Chat') {
-      chat.name = intl.formatMessage(intlMessages.titlePublic);
-    }
-
-    return (
-      <Link
-        to={linkPath}
-        className={cx(styles.chatListItem, linkClasses)}
-        role="button"
-        aria-expanded={isCurrentChat}
-        tabIndex={tabIndex}
-      >
-        <div className={styles.chatListItemLink}>
-          <div className={styles.chatIcon}>
-            {chat.icon ? this.renderChatIcon() : this.renderChatAvatar()}
-          </div>
-          <div className={styles.chatName}>
-            {!compact ? <span className={styles.chatNameMain}>{chat.name}</span> : null }
-          </div>
-          {(chat.unreadCounter > 0) ?
-            <div
-              className={styles.unreadMessages}
-              aria-label={isSingleMessage
-                  ? intl.formatMessage(intlMessages.unreadSingular, { 0: chat.unreadCounter })
-                  : intl.formatMessage(intlMessages.unreadPlural, { 0: chat.unreadCounter })}
-            >
-              <div className={styles.unreadMessagesText} aria-hidden="true">
-                {chat.unreadCounter}
-              </div>
-            </div>
-              : null}
+  return (
+    <Link
+      to={linkPath}
+      className={cx(styles.chatListItem, linkClasses)}
+      role="button"
+      aria-expanded={isCurrentChat}
+      tabIndex={tabIndex}
+    >
+      <div className={styles.chatListItemLink}>
+        <div className={styles.chatIcon}>
+          {chat.icon ?
+            <ChatIcon icon={chat.icon} />
+            :
+            <ChatAvatar
+              isModerator={chat.isModerator}
+              color={chat.color}
+              name={chat.name.toLowerCase().slice(0, 2)}
+            />}
         </div>
-      </Link>
-    );
-  }
-
-  renderChatAvatar() {
-    const user = this.props.chat;
-    return (
-      <UserAvatar
-        moderator={user.isModerator}
-        color={user.color}
-      >
-        {user.name.toLowerCase().slice(0, 2)}
-      </UserAvatar>
-    );
-  }
-
-  renderChatIcon() {
-    return (
-      <div className={styles.chatThumbnail}>
-        <Icon iconName={this.props.chat.icon} className={styles.actionIcon} />
+        <div className={styles.chatName}>
+          {!compact ?
+            <span className={styles.chatNameMain}>
+              {isPublicChat(chat) ? intl.formatMessage(intlMessages.titlePublic) : chat.name}
+            </span> : null}
+        </div>
+        {(chat.unreadCounter > 0) ?
+          <ChatUnreadMessages
+            isSingleMessage={isSingleMessage}
```

Comment:
You dont need this prop since it relies on the unreadCounter. Is there a case where the user has a single message but we want the plural desc?